Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -2,6 +2,45 @@
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
 
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="SystemControlBladeViewForground" Color="{StaticResource SystemBaseHighColor}"/>
```

Comment:
Typo

Reply:
Actually I think what should happen is that the background color should be ignored for High Contrast in both cases. In the non custom color the theme background isn't used in High Contrast so that policy should carry over to specifying a custom color. Can you confirm this is either true or not before I make modifications to the control?

Reply:
I believe the background from the theme should be used. Otherwise, we wouldn't be completely compliant with the High Contrast theme and, if not hurt accessibility, would reduce customization options for the end user.

Reply:
Thanks - I'll make the changes to set the background to be the High Contrast theme one then in both cases.